content/blog/actions-2-0-0/index.Rmd

```diff
@@ -0,0 +1,155 @@
+---
+output: hugodown::hugo_document
+
+slug: actions-2-0-0
+title: GitHub Actions for R developers, v2
+date: 2022-02-14
+author: Gábor Csárdi
+description: >
+    We have updated our GitHub Actions at `r-lib/actions`. Consider upgrading
+    to the new `v2` version, for faster and more reliable GHA jobs.
+
+photo:
+  url: https://www.pexels.com/photo/industry-technology-power-clock-4069389
+  author: Ronaldo Galeano
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [programming] 
+tags: ["GitHub Actions", "r-lib", "actions", "CI", "contionuous integration"]
+---
+
+```{=html}
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+```
+We're tickled pink to announce the new, `v2` versions of our collection of R related GitHub Actions at <https://github.com/r-lib/actions>.
+
+If you are already using these actions, you might want to take look at the [full list of changes](https://github.com/r-lib/actions/releases/tag/v2) first.
+
```

Comment:
Needs a 1-2 sentence summary outlining how the rest of the blog post is going to go.